config/manager/manager.yaml

```diff
@@ -43,17 +43,19 @@ spec:
       containers:
       - command:
         - /root/manager
-        # args:
+        args:
+          - "--auditInterval=30"
           # - "--alsologtostderr"
           # - "--stderrthreshold=INFO"
           # - "-v=100"
-        image: controller:latest
+        image: quay.io/open-policy-agent/gatekeeper:v3.0.4-alpha.0
```

Comment:
If this is meant for building/testing latest from master, this image should be tagged against `latest` so we don't need to update this tag each time we push a new release.

Reply:
That is taken care of by the generated manager_image_patch.yaml generated by the Makefile.

This is meant to be the published release version baked into Gatekeeper.yaml